css/style.css

```diff
@@ -149,24 +142,15 @@ img {
 #info h2 {
     margin: 17px 0 15px 0;
     border: 0;
-    font-family: 'Georgia', 'Ovo', serif;
     color: #2b7dbc;
-    font-size: 1.75em;
+    font-size: 2.2rem;
     font-weight: normal;
 }
 
-#info p {
-  line-height: 1.5;
-}
-
 #content {
-    text-align: justify;
     width: 874px;
     padding-left: 1em;
     padding-right: 1em;
-    line-height: 1.5;
-    color: #fff;
```

Comment:
This removal means we’re using #ddd for content text, which makes things less readable. Can we go back to #fff?

Reply:
The font color change for body text was a deliberate choice. By using #ddd over #2e2e2e we still achieve a contrast ratio of 9.99:1, which is double the minimum contrast guidelines set by the WCAG 2.1 guidelines of 4.5:1 for body text. By using gray for the large blocks of text you also reduce eye strain, and using white for headings with gray body text creates a nice visual hierarchy.

But if you insist we can revert, no sweat.